modules/DMZ/ticket.py

```diff
@@ -0,0 +1,14 @@
+# The ticket is a task that has been assigned to a worker.
+# Each ticket should comprise all of the information needed for the worker to
+# complete the task.
+
+
+class Ticket(object):
```

Comment:
We should probably create an issue and fully flesh out what a ticket means and does